server/src/routes/api/v1/placesRouter.js

```diff
@@ -0,0 +1,10 @@
+import express from "express"
+
+
+const placesRouter = new express.Router()
+
+// placesRouter.post("/", async (req, res) => {
+//   const locations = req.body 
+// })
+
+export default placesRouter
```

Comment:
looks like this one isnt being used yet!

Reply:
I deleted these notes to prevent any really bad code from getting in, but they'll be filled in with the merging of the create-client-form branch